GitUI/UserControls/FileStatusList.Designer.cs

```diff
@@ -50,14 +50,12 @@ private void InitializeComponent()
             this.FileStatusListView.Location = new System.Drawing.Point(0, 21);
             this.FileStatusListView.Margin = new System.Windows.Forms.Padding(3, 4, 3, 4);
             this.FileStatusListView.Name = "FileStatusListView";
-            this.FileStatusListView.OwnerDraw = true;
             this.FileStatusListView.ShowItemToolTips = true;
             this.FileStatusListView.Size = new System.Drawing.Size(682, 464);
             this.FileStatusListView.Sorting = System.Windows.Forms.SortOrder.Ascending;
             this.FileStatusListView.TabIndex = 1;
             this.FileStatusListView.UseCompatibleStateImageBehavior = false;
             this.FileStatusListView.View = System.Windows.Forms.View.Details;
-            this.FileStatusListView.DrawItem += new System.Windows.Forms.DrawListViewItemEventHandler(this.FileStatusListView_DrawItem);
```

Comment:
Has the corresponding method been deleted?

Reply:
Please remove the obsolete handler

Reply:
Thanks for spotting that. I've deleted it (along with a few other methods in that class which weren't being used).